swap_meet/clothing.py

```diff
@@ -1,2 +1,14 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, id=None, condition=0, fabric="Unknown"):
+        super().__init__(id, condition)
```

Comment:
Nice, using the `super` function to get access to the inherited constructor! Making sure to keep the code DRY with a well designed parent class.